cmd/flags.go

```diff
@@ -94,7 +94,13 @@ var (
 	// StaticPeers specifies a set of peers to connect to explicitly.
 	StaticPeers = &cli.StringSliceFlag{
 		Name:  "peer",
-		Usage: "Connect with this peer, this flag may be used multiple times. This peer is recognized as a trusted peer.",
+		Usage: "Connect with this peer and mark it as 'trusted' to ensure connection with it. This flag may be used multiple times.",
+	}
+	// DirectPeers specifies a set of peers to connect to for "Explicit Peering Agreement"
+	// Reference: https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#explicit-peering-agreements
+	DirectPeers = &cli.StringSliceFlag{
```

Comment:
this flag is not necessary. The `-peer` flag means the same thing here as we only specify trusted peers there(those not penalized/disconnected with). Making them a direct peer of our mesh is a logical next step.

Reply:
I see! I added another flag because maybe you wanted to distinguish between "static" and "explicit". For instance, Teku keeps this separation (https://github.com/Consensys/teku/issues/8004#issuecomment-1985928195), while Lighthouse doesn't.

If you want to proceed with making the `-peer` flag configuring direct peers, I'll update my PR accordingly soon!

Reply:
In prysm, this peer is assumed to be trusted: 
https://github.com/prysmaticlabs/prysm/pull/13773/files#diff-e231d920accff9e68d1581fc082513cda53631f2a312c9005448288b29dd08a3R97

So its fine to make them direct too for us. There isn't much benefit to differentiating them with different flags.

Reply:
I've made some changes in order to configure static peers as direct in the pubsub options. I hope it is good now. Thank you!